CHANGELOG.md

```diff
@@ -1,9 +1,16 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Command Generate - Add support for MySQL 8.
+
 ### Fixed
 - Command Generate - Fix error on project generation with database contain table with number as name.
 
+### Changed
+- Tests - Use MySQL 8.0 as the max supported MySQL version.
+- Technical - Update `mysql2` dependency to 2 (latest MAJOR version).
```

Comment:
Same question as in the `forest-express-sequelize` repository:
Is this a necessary change to support MySQL 8?

Reply:
See: https://github.com/ForestAdmin/forest-express-sequelize/pull/373#discussion_r374221991

Reply:
Same response ;)
https://github.com/ForestAdmin/forest-express-sequelize/pull/373#discussion_r374246697